performance/generate.js

```diff
@@ -0,0 +1,24 @@
+const jsdom = require('jsdom');
+const { JSDOM } = jsdom;
+const fs = require('fs');
+const path = require('path');
+
+JSDOM.fromFile(path.join(__dirname, '../demo/vl-button.html')).then(dom => {
```

Comment:
ik zou da precies aan een 2e paginake koppelen ivp aan de standaard demo-pagina?

Reply:
Ik snap niet wat je bedoelt.

Reply:
nu pak je gewoon de 1e uit de demo pagina maar die is niet altijd representatief voor de common case misschien. 
dus ik zou een aparte file voorzien waar je dat per component in moet zetten.

Reply:
Ik dacht dat we besproken hadden op de planning dat dat voorlopig voldoende zou zijn ...

Reply:
dan ben ik niet mee wanneer je voorlopig in niet-voorlopig wil veranderen

Reply:
Ik denk dat we als uitbreiding dit willen runnen voor elke demo block, maar misschien moeten we eerst de setup voor één afronden en kijken hoe performant en vlot alles loopt. Daarna kunnen we uitbreiden. Ik denk dat we al een groot deel coveren door de 1e demo variant te testen. De volgende varianten gaan vaak alleen maar in stijl verschillen, de achterliggende code zou bij de eerste demo al voor een groot stuk gedekt moeten zijn.

@cambiph misschien wel best sub ticket aanmaken om dit niet te vergeten.